<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudioProjectSettingsPage</class>
 <widget class="QWidget" name="StudioProjectSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>269</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Presentation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width x Height</string>
        </property>
        <property name="buddy">
         <cstring>m_ClientSizeWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="m_ClientSizeWidth">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="m_ClientSizeHeight">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="m_checkConstrainProportions">
        <property name="text">
         <string>Constrain Proportions</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="m_checkPortraitFormat">
        <property name="text">
         <string>Portrait Format</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Project Info</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Author</string>
        </property>
        <property name="buddy">
         <cstring>m_Author</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_Author"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Company</string>
        </property>
        <property name="buddy">
         <cstring>m_Company</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_Company"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_ClientSizeWidth</tabstop>
  <tabstop>m_ClientSizeHeight</tabstop>
  <tabstop>m_checkConstrainProportions</tabstop>
  <tabstop>m_checkPortraitFormat</tabstop>
  <tabstop>m_Author</tabstop>
  <tabstop>m_Company</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
